modules/sdk-core/src/bitgo/baseCoin/baseCoin.ts

```diff
@@ -241,15 +241,15 @@ export abstract class BaseCoin implements IBaseCoin {
   /**
    * @deprecated use {@see isWalletAddress} instead
    */
-  verifyAddress(params: VerifyAddressOptions): boolean {
+  verifyAddress(params: VerifyAddressOptions): Promise<boolean> {
     return this.isWalletAddress(params);
   }
 
   /**
    * @param params
    * @return true iff address is a wallet address. Must return false if address is outside wallet.
    */
-  abstract isWalletAddress(params: VerifyAddressOptions): boolean;
+  abstract isWalletAddress(params: VerifyAddressOptions): Promise<boolean>;
```

Comment:
If we don't change this to async here, whenever we create a new coin will this method be made *without* async?

Reply:
One would probably have to add async to the method when implementing but async can't be added to abstract methods.